### PR TITLE
Fix bug in DWTF ancient samples

### PR DESCRIPTION
**Pull request by jeromekelleher**
Can you take a look through please @DomNelson?

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -283,3 +283,45 @@ def test_wf_hudson_single_locus(self):
...
+            random_seed=2)
+        tree = ts.first()
+        self.assertEqual(tree.num_roots, 1)
+        times = ts.tables.nodes.time
+        dtwf_times = times[np.logical_and(times > 0, times < t1, times > t2)]
+        self.assertGreater(dtwf_times.shape[0], 0)
+        self.assertTrue(np.all(dtwf_times == np.floor(dtwf_times)))
+        coalescent_times = times[np.logical_and(times > t1, times < t2)]
+        self.assertGreater(coalescent_times.shape[0], 0)
+        self.assertTrue(np.all(coalescent_times != np.floor(coalescent_times)))
```
Comment (DomNelson) [inline tests/test_models.py]: These look good but don't seem related to ancient samples.

**Comment by DomNelson** [comment]
Looks good to me! Great to have this working

File: tests/test_models.py (hunk 1 above)
Comment (jeromekelleher) [reply tests/test_models.py]: You're right, this was a separate commit where I started working on something else. Didn't seem worth the bother of making another PR for it, so I just kept it in here.

**Comment by jeromekelleher** [comment]
OK, thanks @DomNelson, merging.
